akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala

```diff
@@ -5,21 +5,27 @@
 package akka.pattern
 
 import akka.actor.ActorSystem
-import language.postfixOps
-import scala.concurrent.duration._
-import scala.concurrent.{ Await, ExecutionContext, Future, TimeoutException }
-import scala.util.{ Failure, Success, Try }
 import akka.testkit._
-import org.mockito.ArgumentCaptor
```

Comment:
good! this is the only usage of mockito in akka-actor-tests, so you can remove the dependency there.
Still used in cluster-sharding and cluster-metrics so you might have to add the dependency there instead.